src/CoreShop/Bundle/ResourceBundle/Controller/ResourceSettingsController.php

```diff
@@ -59,8 +59,13 @@ public function getConfigAction(): Response
                 $application = $alias[0];
                 $alias = $alias[1];
 
-                $class = str_replace('Pimcore\\Model\\DataObject\\', '', $definition['classes']['model']);
-                $class = str_replace('\\', '', $class);
+                if (isset($definition['classes']['model'])) {
+                    $class = $definition['classes']['pimcore_class_name'];
```

Comment:
Shouldn't this be `$definition['classes']['model']`?

Reply:
good point, will check again